.github/workflows/ci.yaml

```diff
@@ -35,11 +35,14 @@ jobs:
       fail-fast: false
       matrix:
         # Test modern Odoo versions with latest Postgres version
-        odoo_version: ["17.0"]
+        odoo_version: ["18.0"]
         pg_version: ["15"]
         python_version: ["3.10"]
```

Comment:
I'd say Odoo is supporting oficially 3.12 for this version

Reply:
No, talking with them, they still stick to 3.10 although there are the corresponding changes for 3.12 compatibility